system/lib/libc/emscripten_get_heap_size.c

```diff
@@ -5,8 +5,8 @@
  * found in the LICENSE file.
  */
 
-#include <emscripten/heap.h>
```

Comment:
Why this change?   Normally we want to include the header file where the function we are implement is declared no?

Reply:
I don't recall why this change, it does look accidental now.